tools/github_helper/main.go

```diff
@@ -93,16 +93,42 @@ func newHelper(r *string) (*helper, error) {
 	}
 }
 
+// Create a new branch for fast forward
+func (h helper) createBranchForFastForward(commit *string) error {
```

Comment:
Please pass the branch name as an argument. And generate it in createPullRequestToBase to avoid code duplication